packages/teleport/src/Desktops/DesktopList/DesktopList.tsx

```diff
@@ -206,6 +207,7 @@ type Props = {
   username: string;
   clusterId: string;
   searchValue: string;
+  onLoginMenuOpen: (desktopId: string) => { login: string; url: string }[];
```

Comment:
i think we prefer this syntax (i also think its easier to read)
```suggestion
  onLoginMenuOpen(desktopId: string): { login: string; url: string }[];
```